src/collections/bootstrap.erl

```diff
@@ -398,6 +404,34 @@ col(Page, Block, Width, Offset, Classes) -> % {{{2
        body=common:parallel_block(Page, Block)
       }.
 
+table(Page, Block,Classes) -> % {{{2
+    #table { 
+      html_id=common:block_to_html_id(Block),
+      rows=common:parallel_block(Page, Block),
+      class=Classes
+    }.
+
+tr(Page, Block, Classes) -> % {{{2 
+    #tablerow { 
+      html_id=common:block_to_html_id(Block),
+      class=Classes,
+      cells=common:parallel_block(Page, Block)
+    }.
+
+th(Page, Block, Classes) -> % {{{2 
+    #tableheader { 
+      html_id=common:block_to_html_id(Block),
+      class=Classes,
+      text = common:parallel_block(Page, Block)
+    }.
+
+td(Page, Block, Classes) -> % {{{2 
+    #tablecell { 
+      html_id=common:block_to_html_id(Block),
+      class=Classes,
+      text = common:parallel_block(Page, Block)
```

Comment:
This will cause rendering everything inside cell as escaped text. Use 'body=' instead of 'text='.